extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -0,0 +1,174 @@
+# KHR_draco_mesh_compression
+
+## Contributors
+
+* Fan Zhang, Google, <mailto:zhafang@google.com>
+* Ondrej Stava, Google, <mailto:ostava@google.com>
+* Frank Galligan, Google, <mailto:fgalligan@google.com>
+* Kai Ninomiya, Google, <mailto:kainino@google.com>
+* Patrick Cozzi, Cesium, [@pjcozzi](https://twitter.com/pjcozzi)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a schema to use Draco geometry compression libraries in glTF format. This allows glTF to support streaming compressed geometry data instead of the raw data.
+
+The [conformance](#conformance) section specifies what an implementation must do when encountering this extension, and how the extension interacts with the attributes defined in the base specification.
+
+## glTF Schema Updates
+
+Draco geometry compression library could be used for `primitive` by adding an `extension` property to a primitive, and defining its `KHR_draco_mesh_compression` property.
+
+The following picture shows the structure of the schema update. 
+
+**Figure 1**: Structure of geometry compression extension.
+![](figures/structure.png)
+
+In general, we will use the extension to point to the buffer that contains the compressed data. The major change is that the `accessor` in an extended `primitive` no
+longer points to a `bufferView`. Instead, the `attributes` of a primitive will use the decompressed data. This is valid because in glTF 2.0, `bufferView` is not required in `accessor`, although if it is not present or the id is 0, it will be used with `sparse` field to act as a sparse accessor. In this extension, we will ignore the `bufferView` property.
+
+Usage of the extension must be listed in the `extensionUsed` and `extensionsRequired`. 
+
+```javascript
+"extensionsUsed" : [
+    "KHR_draco_mesh_compression"
+]
+
+"extensionsRequired" : [
+    "KHR_draco_mesh_compression"
+]
+
+```
+
+The extension then could be used like the following, note that all other nodes stay the same
+except `primitives`:
+
+```javascript
+
+"mesh" : {
+    "primitives" : [
+        {
+            "attributes" : {
+                "POSITION" : 11,
+                "NORMAL" : 12,
+                "TEXCOORD_0" : 13,
+            },
```

Comment:
In practical, for an implementation of loader like ThreeJS, it might be OK to ignore the attributes. But it is preferred not to ignore `accessor`s since there are other properties. For example, WebGL might require these properties to render the assets correctly. 
Another reason we don't explicitly omit the `attributes` is that we want to minimize the change in the spec when Draco extension is presented. In our design, the `accessor`s of `attributes` could even have other extensions as long as they use the decompressed data instead of from `bufferView`.